packages/presenters/PostPresenter.js

```diff
@@ -1,33 +1,48 @@
 import { uniq } from 'lodash/fp'
 import { butterflyBush, caribbeanGreen, fakeAlpha, flushOrange, gold, pictonBlue, sunsetOrange } from 'style/colors'
 
+// TODO: Confirm that this presenter is idempotent and reconcile/merge transformations with
+// Mobile PostEditor.store, etc
 export default function PostPresenter (post, { forGroupId } = {}) {
   if (!post) return post
 
-  // if remote exists set url to remote, if not do the following
-  const attachments = post.attachments.map(attachment => {
-    const url = attachment?.url || attachment?.remote
-    return { ...attachment, url, local: attachment.local || url }
-  })
-  const images = attachments.filter(attachment => attachment.type === 'image')
-  const files = attachments.filter(attachment => attachment.type === 'file')
-
   return {
     ...post,
-    attachments,
-    images,
-    files,
-    fileUrls: uniq(files.map(file => file.url)),
-    imageUrls: uniq(images.map(image => image.url)),
+    getAttachments: () => attachmentsResolver(post),
+    getImages: () => imagesResolver(post),
+    getFiles: () => filesResolver(post),
+    getFileUrls: () => uniq(filesResolver(post).map(file => file.url)),
+    getImageUrls: () => uniq(imagesResolver(post).map(image => image.url)),
+    // TODO: Research and reconcile start and endTimeRaw with their incoming values, both are currently
```

Comment:
Yeah, recalling this was related to luxon changes getting brought into shared but not being tested in mobile, and so downstream there were divergent formats of the times needed.